webapp/plugins/vue-filters.js

```diff
@@ -45,9 +46,9 @@ export default ({ app = {} }) => {
       if (length <= 0) {
         return value
       }
-      let output = value.substring(0, length)
+      let output = trunc(value, length).html
       if (output.length < value.length) {
-        output += '…'
+        output += ' …'
```

Comment:
I really love this @mattwr18 ! 💫😍
The ellipse the most and for sure the space as well. 💥